app/assets/stylesheets/components/_top-message.scss

```diff
@@ -0,0 +1,5 @@
+.top-message-size{
+  font-size: .9rem !important;
+  letter-spacing: 0.05rem !important;
+  background-color: var(--black-background) !important;
```

Comment:
- sort alphabetically this block
- remove impotants
- add space in your class with the curly brace
- add a blank line at the end of this file